tests/drivers/can/timing/src/main.c

```diff
@@ -172,7 +172,8 @@ static void test_timing_values(const struct device *dev, const struct can_timing
 	int err;
 
 	printk("testing bitrate %u, sample point %u.%u%% (%s): ",
-		test->bitrate, test->sp / 10, test->sp % 10, test->invalid ? "invalid" : "valid");
```

Comment:
Does it make sense for the tests to _also_ test for -EINVAL separately from -EDOM? It looks like the else block of this conditional would provide that coverage but it's not explicit so I'm just checking here.

Reply:
No CAN controller should return `-EINVAL` here as it is today, but one of the ideas behind a more fine-grained error code result from the CAN bitrate APIs is to be able to test all CiA recommended bitrates and simply ignore the ones where the _bitrate_ is not supported by the given CAN controller (quite a few CAN controllers do not support the low frequency bitrates).